api-server/src/server/boot/authentication.js

```diff
@@ -188,3 +194,53 @@ function createGetPasswordlessAuth(app) {
     );
   };
 }
+
+function mobileLogin(app) {
+  const {
+    models: { User }
+  } = app;
+  return async function getPasswordlessAuth(req, res, next) {
+    const {
+      headers: { token: accessToken }
```

Comment:
```suggestion
      headers: { token }
```
Nit: no need to reassign